test_rclcpp/test/test_sigterm_handler.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+#include <memory>
+
+#include "rclcpp/rclcpp.hpp"
+
+using namespace std::chrono_literals;
+
+void sigtermHandler(int sig)
```

Comment:
@ivanpauno nit: 
```suggestion
void sigterm_handler(int)
```
I know what the GSG says, but we're using snake case in [most cases here](https://docs.ros.org/en/foxy/Contributing/Code-Style-Language-Versions.html#function-and-method-naming).

Reply:
I copy pasted that from `test_signal_handler.cpp`, which by the way should be renamed test_sigint_handler

Reply:
Done, see https://github.com/ros2/system_tests/pull/485/commits/e259da3875ed7e1184a5f82315b95b627dd8c81d.

> I copy pasted that from test_signal_handler.cpp, which by the way should be renamed test_sigint_handler

See https://github.com/ros2/system_tests/pull/485/commits/81b6b6c31189efc08d623a5d9aee0c7fccdc8bc2

Reply:
> 115: /home/ivanpauno/ros2_ws/src/ros2/system_tests/test_rclcpp/test/test_sigterm_handler.cpp:22:  Using C-style cast.  Use static_cast<int>(...) instead  [readability/casting] [4]

cpplint didn't like the unnamed argument :joy:.
Reverted that part in https://github.com/ros2/system_tests/pull/485/commits/25fe274d2488fd63ef17d78ecdf448f9e7454503.